### ISSUE TITLE
Remove unassigned items

### ISSUE THREAD

**Issue by RadiuomOxide** [SharePoint/sp-dev-docs#7749]
Is there any formatting that could hide the unassigned items? My list doesn't allow unassigned items so it's pointless and confusing to users.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 33dfff5b-2b9e-94c7-ffa2-73719f49fb20
* Version Independent ID: 33dfff5b-2b9e-94c7-ffa2-73719f49fb20
* Content: [Format board view to customize SharePoint](https://docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-board-formatting#build-custom-cards)
* Content Source: [docs/declarative-customization/view-board-formatting.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/declarative-customization/view-board-formatting.md)
* Product: **sharepoint**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by shagra-ms** [comment]
@RadiuomOxide Thanks for this. Currently there isn't any way to do this via formatter or any view level configuration. But this is a valid feedback. Will discuss this internally and track the request in our backlog. 

Thanks!

**Comment by bjwo** [comment]
I am facing the same issue - would love to get rid of the "Unassigned items" bucket.

**Comment by neilx1979** [comment]
This issue has been put to Microsoft many times over the years but they've never done anything about it. I really wish they would as "unassigned items" is so annoying when you don't need it (which is most of the time!!!)

**Comment by shagra-ms** [comment]
@neilx1979 not sure if you are referring to the same issue. Board View was relased a few months ago, and the ask here is to be able to hide that.

Can you please elaborate the issue that has been raised so many time over the years?

**Comment by neilx1979** [comment]
The principal is the same as when you get "Unassigned items" in a grouped list.
People have been asking for that to be removed since 2017.

**Comment by Gubbinz117** [comment]
The principal isn't quite the same @neilx1979 as in a grouped list the "Unassigned Items" should only appear if there are unassigned items. In the Board View it's there even if there are none. Get your point though there should be an option to stop grouping by it in grouped views also.

**Comment by JoseVillarreal-CXC** [comment]
If somebody will work on it, it may be worth checking if it's related to more general issue that when the view is filtered (not that there are no items with the value, but filtered) the column in the view stays in (0)

**Comment by bpuncer** [comment]
Any Microsoft follow-up on this topic? What's the point of sharing pain point if there is no update...

**Comment by thearn** [comment]
Agreed that the ability to remove the 'Unassigned items' bucket from the board view would be great. Otherwise, the view is not very useful, based on how much screen space it takes up.

**Comment by darkrob** [comment]
Same for me. Any progress?

**Comment by By-Pixel** [comment]
Waiting for Microsoft to add this and many other fundamental options, in the meantime I use this CSS to hide it:
  div[data-bucket-id='__unassigned__'] {display: none;}

**Comment by lucamanu** [comment]
just stumbled upon this. @By-Pixel's solution is unfortunately unavailable to users without admin rights (or where admin has restricted the possibility to modify CSS). If anyone has found a similar solution using the JSON formatter field, that would be very useful. In the meanwhile: please Microsoft add the possibility to remove this bucket!

**Comment by paulmatheson** [comment]
Any update on this Microsoft?

**Comment by julianascanonisilva** [comment]
Any updates?

**Comment by alinebgf** [comment]
> Waiting for Microsoft to add this and many other fundamental options, in the meantime I use this CSS to hide it: div[data-bucket-id='**unassigned**'] {display: none;}

 Hi all, I know it's been a while now but how can I try this option? I don't understand much of json...tried to include it on json of advanced formating view but didn't work. Please, can someone explain where and how should I add this?
 
 thanks!!

**Comment by lucamanu** [comment]
@alinebgf it's not JSON, it's CSS, and it needs to be added to a manifest file, for which you need to be granted administrative rights. I don't, hence it's not a viable solution.

**Comment by alinebgf** [comment]
@lucamanu ohh ok then... thanks for clarifying! I would never guess it hahaha

Microsoft should really address this, so anoying...

**Comment by justanyusernameplease** [comment]
Please, @shagra-ms, fix this. It's really annoying and makes the board view so much less attractive to use.

**Comment by shagra-ms** [comment]
Thank you everyone for the feedback. I apologise for not being able to get back earlier with an update. The team has been super busy, which didn't allow me to come back with a meaningful update. 

Update on the issue - we are expecting to roll this functionality out this quarter. I hope that addresses your pain points.

Thanks!

**Comment by Svenvdv26** [comment]
@shagra-ms 
Do you have an update on this?
I still see no option to hide/delete it.

**Comment by justanyusernameplease** [comment]
@shagra-ms The bucket "Unassigned" still shows and I can't find an option to hide it. Was this actually rolled out in Q4/2023?

**Comment by aaclage** [comment]
There is a option on buckets that you can hide/show when not needed.

![image](https://github.com/SharePoint/sp-dev-docs/assets/8522348/dacf2539-180f-44c9-82e1-64bf2844693e)

here the bucket option:
![image](https://github.com/SharePoint/sp-dev-docs/assets/8522348/3ac9f0fd-80c2-4e24-bd07-05f288481d82)

**Comment by justanyusernameplease** [comment]
@aaclage Thanks for that. Seems you have an ellipses menu left of the 'plus' icon in the top right corner of each bucket. We don't seem to have that (yet?) in our tenant (tenant in switzerland region).

**Comment by aaclage** [comment]
@justanyusernameplease i have a tenant with activation that allow preview features. 

you can also see the same behavior in Lists online https://lists.live.com/, the feature looks to be there just need to be release globally.

**Comment by a-smiggle** [comment]
Will this become available from within sharepoint? Currently only use this when in Microsoft Lists web app. Once viewed in sharepoint page unassigned is shown again.

**Comment by Svenvdv26** [comment]
> Will this become available from within sharepoint? Currently only use this when in Microsoft Lists web app. Once viewed in sharepoint page unassigned is shown again.
> 
> Then once the sharepoint page is viewed again that view is corrupted.

I have the same issue

**Comment by infamemary** [comment]
Not available for me yet. Tried from the app, from different browsers. Incredibly annoying.

**Comment by rolaseba** [comment]
I have the same issue. Without dots to configure that.

**Comment by shagra-ms** [comment]
@rolaseba this isn't expected. 

Can you share the exact repro?

**Comment by aaclage** [comment]
> Adding my voice to ask for a solution here. I don't have the three dots to configure buckets on the board.

Dear Mr. @joejekpenyong, 

Can you validate if you have the new List UI, you can see in the view bar like sample below as new List UI
![image](https://github.com/user-attachments/assets/1815b75c-3815-4bcf-aaef-06724119b047)
